pkgs/by-name/an/anytype/anytype-heart/default.nix

```diff
@@ -0,0 +1,89 @@
+{
+  stdenv,
+  lib,
+  callPackage,
+  fetchFromGitHub,
+  buildGoModule,
+  protoc-gen-grpc-web,
+  protoc-gen-js,
+  protobuf,
+}:
+
+let
+  tantivy-go = callPackage ../tantivy-go { };
+  pname = "anytype-heart";
+  version = "0.38.9";
+  src = fetchFromGitHub {
+    owner = "anyproto";
+    repo = "anytype-heart";
+    tag = "v${version}";
+    hash = "sha256-0MRtzPSUxbCBJQLJbHsdEpf6GEFoS4ud1S6j9GZWzAE=";
+  };
+
+  arch =
+    {
+      # https://github.com/anyproto/anytype-heart/blob/f33a6b09e9e4e597f8ddf845fc4d6fe2ef335622/pkg/lib/localstore/ftsearch/ftsearchtantivy.go#L3
+      x86_64-linux = "linux-amd64-musl";
+      aarch64-linux = "linux-arm64-musl";
+      x86_64-darwin = "darwin-amd64";
+      aarch64-darwin = "darwin-arm64";
+    }
+    .${stdenv.hostPlatform.system};
```

Comment:
This is missing an or throw otherwise we get a syntax error on other platforms.

Reply:
That is a good point that we missed. We should fix this.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/390170.